Localization/frFR.lua

```diff
@@ -67,19 +67,19 @@ L["dialog.tooltip.ilvl"] = "niveau d'objet requis"
 L["dialog.tooltip.myilvl"] = "mon niveau d'objet"
 L["dialog.tooltip.hlvl"] = "niveau JcJ requis"
 L["dialog.tooltip.pvprating"] = "classement JcJ du chef de groupe"
-L["dialog.tooltip.mprating"] = "classement Mythic+ du chef de groupe"
-L["dialog.tooltip.defeated"] = "nombre de boss de raid vaincu"
-L["dialog.tooltip.members"] = "nombre de membres"
-L["dialog.tooltip.tanks"] = "nombres de tanks"
-L["dialog.tooltip.heals"] = "nombres de soigneurs"
+L["dialog.tooltip.mprating"] = "classement Mythique+ du chef de groupe"
+L["dialog.tooltip.defeated"] = "nombre de boss déjà tué"
+L["dialog.tooltip.members"] = "nombre de membre"
+L["dialog.tooltip.tanks"] = "nombre de tank"
+L["dialog.tooltip.heals"] = "nombre de soigneur"
 L["dialog.tooltip.dps"] = "nombre de DPS"
-L["dialog.tooltip.partyfit"] = "a des places pour les rôles de mon groupe"
+L["dialog.tooltip.partyfit"] = "«y a-t-il des places pour les « rôles » de mon groupe"
 L["dialog.tooltip.classes"] = "nombre de classe spécifique"
 L["dialog.tooltip.age"] = "âge du groupe (en minute)"
-L["dialog.tooltip.voice"] = "a une discussion audio"
+L["dialog.tooltip.voice"] = "y a-t-il une vocal"
 L["dialog.tooltip.myrealm"] = "le chef du groupe est de mon royaume"
 L["dialog.tooltip.noid"] = "instances où je n'ai pas d'ID"
-L["dialog.tooltip.matchingid"] = "groupes avec les mêmes boss tué"
+L["dialog.tooltip.matchingid"] = "même boss tué que mon groupe"
```

Comment:
```suggestion
L["dialog.tooltip.matchingid"] = "même boss tué que moi"
```
Les membres du groupe peuvent avoir différents IDs.